packages/web3torrent/src/utils/torrent-status-checker.test.ts

```diff
@@ -112,41 +114,53 @@ describe('Torrent Status Checker', () => {
     describe('getStatus()', () => {
       it('should return Seeding if the torrent is now Seeding', () => {
         expect(
-          getStatus({
-            createdBy: web3torrent.pseAccount,
-            uploadSpeed: 1000,
-            downloadSpeed: 0,
-            progress: 50,
-            done: false
-          } as Torrent)
+          getStatus(
+            {
+              createdBy: web3torrent.pseAccount,
+              uploadSpeed: 1000,
+              downloadSpeed: 0,
+              progress: 50,
+              done: false
+            } as Torrent,
```

Comment:
In the future, it might be nice to refactor functions like `getStatus` accept as input only data that the function needs to avoid `as` force-casting.